samples/nrf9160/aws_fota/src/main.c

```diff
@@ -438,6 +440,29 @@ void main(void)
 	struct mqtt_client client;
 
 	printk("MQTT AWS Jobs FOTA Sample, version: %s\n", CONFIG_APP_VERSION);
+	printk("Initializing bsdlib\n");
+	err = bsd_init();
```

Comment:
You should include `net/bsdlib.h` and use `bsdlib_init()` instead.

Reply:
fixed